scripts/pre_commit_linter.py

```diff
@@ -1008,6 +1010,25 @@ def _check_docstrings(all_files):
                 if line_num > 0:
                     prev_line = file_content[line_num - 1].lstrip().rstrip()
 
+                # Check for space after """ in the single line docstring.
+                if line.startswith('""" ') and line.endswith('"""'):
+                    failed = True
+                    print'%s --> Line %s: %s' % (
+                        filename, line_num + 1, extra_space_message)
+
+                # Check for space after """ in the multi-line docstring.
+                elif line.endswith('""" '):
```

Comment:
Should we check for the case where it is ` """` (single space before the ending quotes)?

/cc @apb7